daffodil-runtime1/src/main/scala/org/apache/daffodil/dsom/SDE.scala

```diff
@@ -1,224 +1,221 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
```

Comment:
Looks like this file had line endings messed up too.

Reply:
Yes, I corrected and pushed all of them.

Reply:
I think you missed SDE.scala, I still see a huge diff.